go.mod

```diff
@@ -55,7 +55,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.3.3
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/onflow/atree v0.1.0-beta1.0.20211027184039-559ee654ece9
-	github.com/onflow/cadence v0.20.0-beta10
+	github.com/onflow/cadence v0.20.0
```

Comment:
We might want to use `v0.20.1` due to [the API mismatch](https://github.com/onflow/cadence/pull/1266) we had in `v0.20.0`